src/System.Memory/tests/Span/CopyTo.cs

```diff
@@ -34,6 +34,24 @@ public static void TryCopyToSingle()
         }
 
         [Fact]
+        public static void TryCopyToDefaultImplicit()
+        {
+            int[] dst = default;
+            Span<int> srcSpan = default;
+            bool success = srcSpan.TryCopyTo(dst);
+            Assert.True(success);
+        }
+
+        [Fact]
+        public static void TryCopyToNullImplicit()
+        {
+            int[] dst = null;
+            Span<int> srcSpan = default;
+            bool success = srcSpan.TryCopyTo(dst);
+            Assert.True(success);
+        }
```

Comment:
How is this test different from the previous one? We don't need a rest that verifies that the c# compiler knows that default for an array is null.